COMMITTERS.md

```diff
@@ -34,6 +35,10 @@ This file details what is needed for different operations in the repository that
 
 Before providing a fixed package version, you need to determine the existing version specified in the repository, what type of version it uses, and whether the package contains additional handling for creating fixed versions.
 
+### Updating a Manual Package
+
+A manual package is one that is not using the `Chocolatey-AU` automation. The specific package files should be updated normally (i.e. the `.nuspec` file and the Chocolatey package scripts). Once those are updated, and the pull request merged, the package is pushed to the Chocolatey Community Repository by making an empty commit, with the message in the format `[PUSH <PACKAGENAME>]` where `<PACKAGENAME>` is the name of the package you want to push.
+
```

Comment:
I'm not sure if it makes sense in this PR specifically, but since we discussed it today I thought I would drop it here as a suggestion. If we want to do some more wordsmithing of it, I'm more than happy to open my own PR for this specific information.

```suggestion

If the manual package is an embedded, or semi-embedded package, the mechanism for updating the files in the package is an `update.ps1` script beside the `.nuspec` file. This file will be run when the package is pushed, but files will not be committed back to the repository.

```